_posts/2017-12-01-ome-files-0-5-0.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Release of OME Files C++ 0.5.0
+intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.5.0
+---
+OME Files is a reference implementation of the OME Data Model and
+OME-TIFF file format for the storage and interchange of biological
+imaging data and metadata.  It provides support for:
+
+- the OME Data Model (model objects, metadata store, XSL transforms
+  and XML validation)
+- OME-TIFF reading
+- OME-TIFF writing
+
+The main change in this release is the re-addition of compatibility
+code to permit building on CentOS6 and RHEL6 with EPEL.  Other work
+includes the switch to C++11 threads and mutexes from the Boost
+equivalents, and preliminary CMake support for building with C++17
+compilers.
+
+**Downloads**
+
+The software release is available at [archived downloads](https://downloads.openmicroscopy.org/ome-files-cpp/0.5.0/). This
+page includes links to the individual repositories and source
+releases, as well as the documentation and binary builds of the
+collection for several common platforms, including Windows, Linux and
+MacOS X.
+
+**Upgrading from 0.4.x**
+
+- No changes required.
+
+**Changes since OME Files C++ 0.3.2**
+
+*All components*
+
+- Use CMP0067 to enable standard setting in feature tests
+- CMake 3.4 is the minimum version
+- Use C++11 `<thread>` and `<mutex>`
+- Restore support for Boost 1.53
+- Add support for Boost 1.65 and 1.65.1
+- Drop embedded copies of CMake `FindBoost`, `FindXalanC` and `FindXercesC`
+- Only run thread tests with CMake ≥3.8
+
+*OME Common*
+
+- Conditionally add `Boost::log` and `Boost::log_setup` to exported configuration
+
+*OME Model*
+
+- Add missing explicit `<sstream>` include
+- Remove REQUIRED from exported cmake configuration
+
+*OME Files*
+
+- Add tiling performance benchmark documentation
+- Add missing implicit `<cstdlib>` and `<iostream>` includes
+- Remove REQUIRED from exported cmake configuration
+
+*OME QtWidgets*
+
+- Remove REQUIRED from exported cmake configuration
+- Add Dockerfile for building with Travis
+
+*Super-Build 0.4.0*
+
+- Windows batch files are always checked out with CRLF line endings
+- Support building with EPEL packages (CentOS and RHEL 6 and 7)
+- Download required Python build dependencies into a virtualenv
+  rather than downloading and installing individually
+- Package changes:
+  - ome-common 5.5.0: Updated package
+  - ome-model 5.6.0: Updated package
+  - ome-files 0.5.0: Updated package
+  - ome-files-performance: New experimental package (git only)
+  - ome-qtwidgets 5.4.3: Updated package
+  - boost 1.65.1: Updated package
+  - png 1.6.34: Updated package
+  - pybind11 2.2.1: New package
+  - tiff 4.0.9: Updated package
+  - yaml-cpp 0.5.3: New package
+  - all python-* packages are now installed with pip
+
+**Errata**
+
+See the
+[errata](https://docs.openmicroscopy.org/ome-cmake-superbuild/0.5.0/errata.html)
```

Comment:
This link should be http://docs.openmicroscopy.org/ome-files-cpp/0.5.0/ome-cmake-superbuild/manual/html/errata.html

Reply:
Fixed.